ha/core/cluster/cluster_manager.py

```diff
@@ -66,7 +67,7 @@ def __init__(self):
         self._execute = SimpleCommand()
         self._decision_monitor = DecisionMonitor()
         # TODO: add node_manager class to handle query
-        self._refresh_contex = PcsRefreshContex(self._decision_monitor)
```

Comment:
No change needed for class PcsClusterManager(ClusterManager):
in main branch

in main branch CortxClusterManager is used

Reply:
Done

Reply:
Done